src/test/ui/impl-trait/issue-60473.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0720]: opaque type expands to a recursive type
```

Comment:
This doesn't look correct to me, please leave a FIXME to revaluate.
Also, cc @matthewjasper re. ^---. Is this is due to some hack?